ethexe/runtime/common/src/transitions.rs

```diff
@@ -55,14 +59,14 @@ impl InBlockTransitions {
     }
 
     pub fn state_of(&self, actor_id: &ActorId) -> Option<H256> {
```

Comment:
shouldn't this return StateHashWithQueue?

Reply:
Fixed